### DISCUSSION TITLE
Media Browser Panel Improvements

### DISCUSSION THREAD

**Discussion by hunterjm** [home-assistant/frontend#7344] [Other feature requests]
- Should Web Browser player allow switching to next/previous item when opened using arrow keys or clicking on the sides of the image for navigation?
- Should media class for images and videos display in a 16:9 aspect ratio instead of having images cropped?

![image](https://user-images.githubusercontent.com/747670/96007503-87763700-0e0c-11eb-87a2-30f14cab5ffe.png)

**Comment by SeanPM5** [comment]
Yeah 16:9 aspect ratio would be very nice to have here I think. That is how those Xbox game clips are displayed on the console and their mobile apps.

Here's a comparison of how Home Assistant and Plex displays the same info. The way the thumbnails get cropped in the HA media browser leaves out a lot of details, look at the sixth thumbnail especially - you can only see part of an arm and not the 3 people in the pic.... Pretty much all the others have people cropped out as well because you lose the left and right sides.

![image](https://i.imgur.com/sUvuHsj.jpg)

**Comment by matthiasdebaat** [comment]
> * Should media class for images and videos display in a 16:9 aspect ratio instead of having images cropped?

Yes, good suggestion. @hunterjm is still in the current HA version?

**Comment by matthiasdebaat** [comment]
> * Should Web Browser player allow switching to next/previous item when opened using arrow keys or clicking on the sides of the image for navigation?

Do you mean for this type of card? That I wouldn't recommend, because of other keyboard shortcuts. Clicking on the buttons below the image is more intuitive from my point of view.
<img width="340" alt="Schermafbeelding 2021-08-23 om 13 59 01" src="https://user-images.githubusercontent.com/20203284/130443376-4d31d915-d3f6-49bd-aa43-4d6ca99e6e41.png">
